pkg/services/datasources/service/datasource_service.go

```diff
@@ -158,7 +147,12 @@ func (s *Service) GetDataSourcesByType(ctx context.Context, query *models.GetDat
 
 func (s *Service) AddDataSource(ctx context.Context, cmd *models.AddDataSourceCommand) error {
 	var err error
-	cmd.EncryptedSecureJsonData, err = s.SecretsService.EncryptJsonData(ctx, cmd.SecureJsonData, secrets.WithoutScope())
+	secret, err := json.Marshal(cmd.SecureJsonData)
```

Comment:
Might be worth adding the secret after the datasource has successfully been created.

Reply:
Yup I noticed that during the demo.

Reply:
Done on 145cac0